certified-connectors/touchSMS/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Access Token",
+          "description": "The access token from your touchSMS account",
+          "tooltip": "Provide the Access Token",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Token ID",
+          "description": "The Token ID  from your touchSMS account",
+          "tooltip": "Provide the Token ID",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
```

Comment:
please make sure you are using the iconBrandColor for your connector. Icon color must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).

Reply:
This was resolved in the commit done yesterday